Description: Use debian/dirs instead of install-data-local
 Replace ugly upstream hack with debian/dirs
 .
 swapspace (1.12-1) unstable; urgency=low
 .
   * Use PREFIX variables when necessary
   * Account for normal install (i.e. /usr/local) in code and initscripts
   * Clean up Makefile flags
   * Add full text of GPL-2
   * Compile hog
Author: Jacob Adams <tookmund@gmail.com>

---

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,5 +1,7 @@
 SUBDIRS = src
 sysconf_DATA = swapspace.conf
 
-install-data-local:
-	$(MKDIR_P) $(localstatedir)/lib/swapspace
+#Hack uneccessary in package; using debian/dirs instead
+#install-data-local:
+#	$(MKDIR_P) $(localstatedir)/lib/swapspace
+
